code-challenges.js

```diff
@@ -9,31 +13,133 @@
 
 // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 42
-// var temp = 350
-// var temp = 212
-
+const boilingPoint = (temperature) => {
+  if (temperature > 212) {
+    return `${temperature} is above boiling point`;
+  } else if (temperature < 212) {
+    return `${temperature} is below boiling point`;
+  } else {
+    return `${temperature} is at boiling point`;
+  }
+};
+
+// var temp = 42;
+// var temp = 350;
+var temp = 212;
+console.log(boilingPoint(temp));
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
-var myNumbers1 = [3, 7, 0, 36, -9]
-var myNumbers2 = [8, -7, 42, 9, 13]
+const lengthOfCombinedArrays = (array1, array2) => {
+  var arraysCombined = array1.concat(array2);
+  return arraysCombined.length;
+};
```

Comment:
Just as a fun refactor, you can chain the length attribute on the end of your concat and just return the one line.